test/shopify-cli/command/project_command_test.rb

```diff
@@ -0,0 +1,46 @@
+require "test_helper"
+
+module ShopifyCLI
+  class Command
+    class FakeVisibleSubCommand < ShopifyCLI::Command::SubCommand; end
+
+    class FakeHiddenSubCommand < ShopifyCLI::Command::SubCommand
+      hidden_feature
+    end
+
+    class FakeCommand < ShopifyCLI::Command::ProjectCommand
+      subcommand :FakeVisibleSubCommand, "fake_visible"
+      subcommand :FakeHiddenSubCommand, "fake_hidden"
+
+      def self.name
+        "fake"
+      end
+
+      def self.messages
+        {
+          fake: {
+            help: "Usage: {{command:%1$s fake [ %2$s ]}}",
+          },
+        }
+      end
+    end
```

Comment:
I generally define constants nested in the test class, so that the global namespace is not polluted. Would that work here?

Reply:
Yes! Done